README.md

```diff
@@ -148,8 +148,13 @@ info['target'] = target
 ## Contributing & Getting Help
 This project is just getting started. We were so eager to get pyxet out that we have not gotten all the code over to this repository yet. We will bring the code here very soon. We fully intend to develop this package in public under the BSD license. 
 
-In the coming days we will add a roadmap to make it easier to know when pyxet features are being implemented and how you can help.
-
 For now, join our [Discord server](https://discord.gg/KCzmjDaDdC) to talk with us. We have ambitious plans and some very useful features under development / partially working (ex. write back to XetHub repos, easy commit messages, stream repositories locally, easily load the same file across Git branches, and more).
```

Comment:
The pyxet roadmap kinda covers line 151, maybe just update that with the Discord link and remove 151.

 ## pyxet Roadmap
 The open source community is very important to us at XetHub and we strive for transparency in sharing our plans for the future. We welcome feedback — join us on [Discord](https://discord.gg/KCzmjDaDdC) to help shape our roadmap!